Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -89,3 +91,54 @@
     line height remains unchanged. */
   content: "\200b";
 }
+
+.bocEnumValue input[type=radio]
+{
+  z-index: var(--z-index-hidden);
+  position: absolute;
+  opacity: 0;
+}
+
+.bocEnumValue input[type=radio] + label
+{
+  margin-bottom: 0.125em;
+}
+
+.bocEnumValue input[type=radio] + label::before
+{
+  background-image: url('../Image/sprite.svg#RadioButtonFalse');
+  height: var(--default-icon-size);
+  width: var(--default-icon-size);
+  display: inline-block;
+  content: "";
+  margin-bottom: -0.25em;
+  margin-right: 0.25em;
+}
+
+.bocEnumValue input[type=radio]:checked + label::before
+{
+  background-image: url('../Image/sprite.svg#RadioButtonTrue');
+}
+
+.bocEnumValue input[type=radio]:disabled + label:before
```

Comment:
please check for consistency, both in the new code and in the existing code regarding before/after